hooks/honeybadger/payload.go

```diff
@@ -0,0 +1,48 @@
+package logrus_honeybadger
+
+import "os"
+
+type Payload struct {
+	Notifier *Notifier `json:"notifier"`
+	Error    *Error    `json:"error"`
+	Server   *Server   `json:"server"`
+}
+
+type Notifier struct {
+	Name    string `json:"name"`
+	URL     string `json:"url"`
+	Version string `json:"version"`
+}
+
+type Error struct {
+	Backtrace []*Backtrace `json:"backtrace"`
+	Message   string       `json:"message"`
+}
+
+// Not used for now
+type Backtrace struct {
+	Number string `json:"number"`
+	File   string `json:"file"`
+}
+
+type Server struct {
+	EnvironmentName string `json:"environment_name"`
+	Hostname        string `json:"hostname"`
+}
+
+func NewPayload(message, env string) (*Payload, error) {
+	hostname, err := os.Hostname()
```

Comment:
hostnames arguably don't change during the process' lifetime, it's safe to save the value in the client